1-exercises/E-for-loop/exercise2.js

```diff
@@ -26,6 +26,12 @@ const AGES = [
     49
 ];
 
+for ( let i = 0; i < 5; i++){
+    console.log(WRITERS[i] + " is " + AGES[i] + " years old");
```

Comment:
String interpolation 
console.log(`${WRITERS[i]} is ${AGES[i]} years old`) ; 
can be used here as well